.editorconfig

```diff
@@ -1,20 +1,21 @@
+
 root = true
 
 [*]
 charset = utf-8
 end_of_line = lf
 
+
 [*.java]
 indent_style = tab
-tab_width = 4
-trim_trailing_whitespace = true
 insert_final_newline = true
-ij_continuation_indent_size = 8
-max_line_length = 120
+trim_trailing_whitespace = true
```

Comment:
I see you removed: `tab_width = 4` and `ij_continuation_indent_size = 8`. I think I had to add both to make the continuation indentation work in IntelliJ (it got translated to 1 tab for normal indentation and 2 tabs for continuation indentation).

Reply:
I think defining the tab size defeats the purpose of tabs, because one could make the indentation larger or narrower without changing the style. I haven't noticed any problems in IntelliJ, and we used a similar config for a long time.
Maybe it also has to do with the “smart tabs” features (https://www.jetbrains.com/help/idea/code-style-sql-tabs-and-indents.html) but I'm not sure.

Reply:
I tried the new settings out and the continuation indentation works fine. For some reason, a few years ago, I had to add these two settings as a workaround. There is even issues reported for that:
- https://youtrack.jetbrains.com/issue/IDEA-266666/editorconfig-messing-up-javas-continuation-indent-of-long-lines (Java, older one)
- https://youtrack.jetbrains.com/issue/IDEA-254520/Continuation-indent-should-not-be-overriden-by-editorconfig (Kotlin, recent one).

Reply:
> I think defining the tab size defeats the purpose of tabs, because one could make the indentation larger or narrower without changing the style. 

@rakow Actually, not specifying `tab_width` and at the same time specifying `max_line_length` is also inconsistent. For instance, if we have 3 tabs of indentation at the beginning of a line, should we break the line after 126, 138 or 144 characters (tab sizes: 8, 4 and 2). I think we should declare the tab size to avoid different behaviours across different IDEs.